check-eve.py

```diff
@@ -36,7 +36,8 @@ def validate_json(args, json_filename, schema):
     errors = []
 
     if not args.python_validator:
-        cp = subprocess.run(["eve-validator", "-q", "-s", schema, "--", json_filename])
+        progname = os.path.join(TOPDIR, "eve-validator", "target", "release", "eve-validator")
```

Comment:
That will not work in my setup where I have `export CARGO_BUILD_TARGET="x86_64-apple-darwin"`

Reply:
Are you cross compiling?

Reply:
Would it be OK if the script cleared that var first?  This is being built to run on the host that is doing the building, so I don't see a case where a custom target would be required.

Reply:
> Are you cross compiling?

No, but (IIRC) this is required by Suricata fuzzing (to rebuild std with sanitizers)

> Would it be OK if the script cleared that var first? This is being built to run on the host that is doing the building, so I don't see a case where a custom target would be required.

For me, I guess so.

But my point was rather that I do not know how to get the build directory out of rust in every case...

Reply:
Could just use this var in the target directory.  I don't think I've ever seen a case where the `--target` arg or var resulted in an output path that didn't match.

Reply:
If this env var is the only way to modify it, I am ok... But it may be in some cargo configuration file as well, do you know ?